packages/lit-query/src/QueryClientProvider.ts

```
@@ -0,0 +1,108 @@
+import { ContextProvider } from '@lit/context'
+import type { QueryClient } from '@tanstack/query-core'
+import type { TemplateResult } from 'lit'
+import { LitElement, html } from 'lit'
```

ESLint

[error] `lit` import should occur before type import of `@tanstack/query-core`

(import/order)